### PR TITLE
Remove deprecated resource [goes with v1.52.0]

### PR DESCRIPTION
**Pull request by BShrivastav**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/vpc/resource_ibm_is_vpc_address_prefix.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ const (
 	isVPCAddressPrefixHasSubnets = "has_subnets"
 	isVPCAddressPrefixDefault    = "is_default"
 	isAddressPrefix              = "address_prefix"
+	isVPCRouteDestinationCIDR    = "destination"
```
Comment (SunithaGudisagarIBM) [inline ibm/service/vpc/resource_ibm_is_vpc_address_prefix.go:24]: isVPCRouteDestinationCIDR field is not used any where except in the validation and that validation identifier is "destination" and "destination" field is not in the schema. Can you please help me to understand this?

**Review by SunithaGudisagarIBM — CHANGES_REQUESTED** [review]
Hey, Can you check my review comment please

File: ibm/service/vpc/resource_ibm_is_vpc_address_prefix.go (hunk 1 above)
Comment (BShrivastav) [reply ibm/service/vpc/resource_ibm_is_vpc_address_prefix.go:24]: This is an existing limitation in the code, current changes removes the deprecated resource/ds as part of same the declaration of the `destination` variable `isVPCRouteDestinationCIDR` is moved from the removed to used one.

We can resolve this incomplete usage of the variable, with current changes.

**Comment by deepaksibm** [comment]
also remove examples from examples/ibm-is-ng/main.tf

**Review by SunithaGudisagarIBM — APPROVED** [review]
Hi Bhavesh, 

PR looks good to me, Can you please create PR for master branch.

Thank you
Sunitha

**Comment by BShrivastav** [comment]
merged: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4496

**Pull request closed without merging**